packages/docregistry/src/context.ts

```diff
@@ -496,7 +505,20 @@ export class Context<T extends DocumentRegistry.IModel>
         const name = PathExt.basename(localPath);
         this._handleError(err, `File Save Error for ${name}`);
         throw err;
-      });
+      })
+      .then(
+        value => {
+          // Capture all sucess paths and emit completion
```

Comment:
`sucess` => `success` ... also, please could you format comments in full sentences (including periods at the end)? Thank you!